packages/medusa/src/services/payment-provider.ts

```diff
@@ -809,6 +809,7 @@ export default class PaymentProviderService extends TransactionBaseService {
 
       const toCreate = {
         payment_id: payment.id,
+        order_id: payment.order_id,
```

Comment:
note: associate created refund with the order otherwise creating refund with `/admin/payments/:id/refund` will create "orphan" refunds that are not calculated in order totals

Reply:
Can I get you to create a ticket for this, so we won't forget?

Reply:
this line should be a fix for the issue, I just added a comment to explain why I made this change

Reply:
Ah sorry, I misunderstood that. Thought you meant we need the link.